src/Aquifer.Jobs/Subscribers/TranslationMessageSubscriber.cs

```diff
@@ -245,7 +245,8 @@ public async Task UpdateProjectPostTranslationAsync([ActivityTrigger] UpdateProj
             .Where(rcv => rcv.ResourceContent.ProjectResourceContents.Any(prc => prc.Project.Id == project.Id) &&
                 rcv.IsDraft &&
                 (rcv.ResourceContent.Status == ResourceContentStatus.TranslationAiDraftComplete ||
-                    rcv.ResourceContent.Status == ResourceContentStatus.AquiferizeAiDraftComplete) &&
+                    rcv.ResourceContent.Status == ResourceContentStatus.AquiferizeAiDraftComplete ||
+                    rcv.ResourceContent.Status == ResourceContentStatus.New) &&
```

Comment:
Adding `New` here ensures that already translated items that were skipped during translation are still assigned upon project start.